### ISSUE TITLE
Questions about pretrained-box, hierarchy loss and reload

### ISSUE THREAD

**Issue by ShingHuuu** [yasumasaonoe/Box4Types#2]
Hi, i have 3 questions in code

1. in training process, i find args named  "pretrained_box_path", is this the model trained after training process?  but I suppose the output of training process is fine-tuned box, so i don't know the exact meaning of pretrained box 
2. In paper, i saw no hierarchy loss  in this project, but in code, i find hierarchy loss in the definition of model, what is this hierarchy loss?
3. For args of training, what is the difference between reload and load 

thanks a lot

**Comment by yasumasaonoe** [comment]
Hi, thanks for your questions! I keep some args that I used during exploratory experiments. So, "pretrained_box_path" and "hierarchy loss" are not used for the final experiments. 

1. pretrained_box_path: This is used to initialize boxes with pertained box parameters (e.g., boxes trained with other  methods).
2. hierarchy loss: Our typing model treat each type box independently (i.e., ignoring hierarchies such as Person > Artist). I used this loss to inject hierarchies between type boxes.  
3. "load" is a boolean, and a model parameters will loaded from a checkpoint if this is True. "reload_model_name" is str and specifies the model file name.
